Update Docs and Fix a big on the onCompleteMerging (#192)
nf-core/multiplesequencealign: docs/output.md

The main output of this pipeline are the **computed MSAs** and **summary reports**, where all the evaluation metrics and resources used are collected. .

Additionally, the pipeline can provide a variety of files useful to inspect (e.g. used guide trees).

The directories listed below will be created in the results directory (specified by `--outdir`) after the pipeline has finished.

All paths are relative to the top-level results directory.

- `results/`

  - <details markdown="1">
    <summary><code>alignments/</code></summary>
      <b>MSA computed.</b>

    Each subdirectory is named after the sample id. It contains all the computed alignments for the given sample. The filename is built with the informations of the input file used and the tool(s).

    - <code>{SampleID}/{SampleID}_{Tree}\_args-{Tree_args}_{MSA}\_args-{MSA_args}.aln</code>.

    </details>

  - <details markdown="1">
    <summary><code>trees/</code></summary>

    <b>Rendered guide trees </b>

    If you have explicitly enabled the computation of guide trees via the toolsheet to be used by the MSA tool, these guide trees will be generated and stored in the trees directory.

>     Each subdirectory is named after the sample id. It contains all the computed trees for the given sample. The filename is built with the informations of the input file used and the tool(s).

Reviewer: ```suggestion
    Each subdirectory is named after the sample id and contains all the computed trees for the given sample. The filename is built with the information from the input file used and the tool(s).
```
